Resources/Private/Partials/ActionGroupDropdowns.html

```
@@ -0,0 +1,44 @@
+<html data-namespace-typo3-fluid="true"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)